.reuse/dep5

```diff
@@ -0,0 +1,17 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: A3 Audio
+Upstream-Contact: A3 Audio <contact@a3-audio.com>
+Source: https://github.com/a3-audio/a3-core
+Comment: https://a3-audio.com
+
+Files: docs/*
+Copyright: 2023 A3 Audio <contact@a3-audio.com>
+License: CC-BY-SA-4.0
+
+Files: platform-config/*
+Copyright: 2023 A3 Audio <contact@a3-audio.com>
+License: CC-BY-SA-4.0
```

Comment:
Should we share these as CC0 as well? Otherwise, technically speaking, if someone sends the patchbay files around or modifies them, he would have to keep an attribution notice around etc.